pkg/cnab/cnab-to-oci/registry.go

```diff
@@ -197,7 +196,7 @@ func (r *Registry) IsImageCached(ctx context.Context, invocationImage string) (b
 
 	imageSummaries, err := cli.Client().ImageList(ctx, imageListOpts)
 	if err != nil {
-		return false, errors.Wrapf(err, "could not list images")
+		return false, fmt.Errorf("not list images: %w", err)
```

Comment:
oops, lost a word here

```suggestion
		return false, fmt.Errorf("could not list images: %w", err)
```